app/main.py

```diff
@@ -1,8 +1,36 @@
 class Car:
-    # write your code here
-    pass
+    def __init__(self, comfort_class: int, clean_mark: int, brand: str) -> None:
+        self.comfort_class = comfort_class
+        self.clean_mark = clean_mark
+        self.brand = brand
 
 
 class CarWashStation:
-    # write your code here
-    pass
+    def __init__(self, distance_from_city_center: float, clean_power: int,
+                 average_rating: float, count_of_ratings: int) -> None:
+        self.distance_from_city_center = distance_from_city_center
+        self.clean_power = clean_power
+        self.average_rating = round(average_rating, 1)
+        self.count_of_ratings = count_of_ratings
+
+    def calculate_washing_price(self, car_cal: Car):
+        price = car_cal.comfort_class * (self.clean_power - car_cal.clean_mark)
+        price *= self.average_rating / self.distance_from_city_center
+        return round(price, 1)
+
+    def wash_single_car(self, car_wash: Car):
```

Comment:
rename the variables for better understanding its purpose. Here it can be simply `dirty_car`